### ISSUE TITLE
Check results are not accessible for screen reader users

### ISSUE THREAD

**Issue by 00travelgirl00** [boswall/Content-Author-Accessibility-Preview#7]
currently the check results are displayed only visually. In this way Screenreader Users can't use the checker. It would be great if the plugin would also be accessible for blind users, because some accessibility consultants are blind. 

I have noticed that problem because it got mentioned in a newsletter from a blind accessibility consultant.

**Comment by boswall** [comment]
The result labels are inserted into the page with JavaScript, so they should be visible to screenreaders. Although I have not tested this with NVDA yet.

Perhaps screenreaders are not reading elements that are visually hidden by a 2D transform?? The show on hover effect does this.

I'm going to either remove the hover effect or have it as an option in the next revision.

**Comment by 00travelgirl00** [comment]
@boswall I have asked him again, if he could give more details about the issue and he checked it and said he can actually see it with a screenreader. He were just assume it, that it was only visual.

So it was a wrong issue. Sorry about this. 

Here is the Tweet about it (in german): 
https://twitter.com/domingos2/status/1157536341447651328
